packages/server/modules/user-emails/repositories/userEmails.ts

```diff
@@ -0,0 +1,42 @@
+import { Knex } from 'knex'
+import crs from 'crypto-random-string'
+import { CreateUserEmail, DeleteUserEmail, UpdateUserEmail } from '../domain/operations'
+import { UserEmail } from '../domain/types'
```

Comment:
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#3387742436de414082cd4e9607259ad9

fix this in all files please. VSCode should be configured to auto-import absolute paths too, so check your config